go/src/koding/kites/kloud/provider/do/schema_test.go

```diff
@@ -0,0 +1,97 @@
+package do
+
+import "testing"
+
+func TestCredential_Valid(t *testing.T) {
+	cred := &Credential{
```

Comment:
Another (small) note: all tests in this file could be merged into one with table style fashion.